docusaurus/docs/extensions/extensions-getting-started.md

```diff
@@ -12,23 +12,20 @@ You'll also need the yarn package manager installed, which can be done with `npm
 
 ## Creating the Application
 
-In order to develop a new Extension, you need an application UI to host it in during development. Rancher provides a helper to create a skeleton application for you. This
-gives you a full version of the Rancher UI that can be used to develop and test your extension.
+To develop a new extension, you need an application UI to host it during development. Rancher provides a helper to create a skeleton application for you. This gives you a full version of the Rancher UI that can be used to develop and test your extension.
 
-Rancher publishes two npm packages to help bootstrap the creation of the app and an extension. These are used in this example below.
+Rancher now publishes a single npm package, `@rancher/extension`, to help bootstrap the creation of the application and extension. This replaces the previous separate creators (`@rancher/app` and `@rancher/pkg`).
 
-### Creating the Skeleton App
+### Creating an Extension
 
 Create a new folder and run:
 
 ```sh
-yarn create @rancher/app my-app [OPTIONS]
+yarn create @rancher/extension my-app [OPTIONS]
 cd my-app
 ```
 
-This will create a new folder `my-app` and populate it with the minimum files needed.
-
-> Note: If you don't want to create a new folder, but instead want the files created in an existing folder, use `yarn create @rancher/app .`
+This command will create a new folder `my-app` and populate it with the minimum files needed.
```

Comment:
This will create a folder called `my-app` and an extension called `my-app`, right? The main argument is for the extension, then you would have an option to customise the app name `-a <name>` correct?

Imo, I would say that it would create a folder called `my-app` and an extension called `my-app` on this part